src/data/userData.ts

```diff
@@ -83,22 +83,23 @@ export async function updateUser(
     if (updatedFields.email) {
       await adminAuth.updateUser(id, { email: updatedFields.email });
     }
-  } catch (pingus) {
-    throw new Error(`Failed to update user: ${pingus}`);
+  } catch (error) {
+    throw new Error(`Failed to update user: ${error}`);
```

Comment:
last comment. these should probably be `error.message` since it is still nesting errors, if you are catching a real error. 

this is probs also throwing typescript errors - caused by error type being unknown: either being an error object or string. I created a logError() function in the util/helper.ts file. it does the typescript check and you can just pass in the extra message you want to put in front of the error like "failed to update user". I used it in the seed.ts file